src/main/java/com/esri/samples/na/routing_around_barriers/RoutingAroundBarriersSample.java

```diff
@@ -29,35 +29,35 @@
   @Override
   public void start(Stage stage) throws Exception {
 
-      // set up the scene
-      FXMLLoader loader = new FXMLLoader(getClass().getResource("/fxml/routing_around_barriers.fxml"));
-      Parent root = loader.load();
-      controller = loader.getController();
-      Scene scene = new Scene(root);
-
-      // set title, size and add scene to stage
-      stage.setTitle("Routing Around Barriers Sample");
-      stage.setWidth(800);
-      stage.setHeight(700);
-      stage.setScene(scene);
-      stage.show();
-    }
-
-    /**
-     * Stops and releases all resources used in application.
-     */
-    @Override
-    public void stop () { controller.terminate(); }
-
-    /**
-     * Opens and runs application.
-     *
-     * @param args arguments passed to this application
-     */
-    public static void main (String[]args){
-
-      Application.launch(args);
-    }
+    // set up the scene
+    FXMLLoader loader = new FXMLLoader(getClass().getResource("/fxml/routing_around_barriers.fxml"));
+    Parent root = loader.load();
+    controller = loader.getController();
+    Scene scene = new Scene(root);
+
+    // set title, size and add scene to stage
+    stage.setTitle("Routing Around Barriers Sample");
+    stage.setWidth(800);
+    stage.setHeight(700);
+    stage.setScene(scene);
+    stage.show();
   }
 
+  /**
+   * Stops and releases all resources used in application.
+   */
+  @Override
+  public void stop () { controller.terminate(); }
```

Comment:
we should follow normal convention here with a new line and correct indentation for methods, like this:

```
public void stop () {
  controller.terminate();
}
```